packages/client/src/providers/AuthProvider/index.js

```diff
@@ -0,0 +1,74 @@
+import React, { useState, useEffect, useContext, createContext } from 'react';
+import * as firebase from 'firebase/app';
+import 'firebase/auth';
+
+const firebaseConfig = {
+    apiKey: 'AIzaSyCSyjSnzA5mp8Y3ro904vg_wxMsG3ELsek',
+    authDomain: 'coved-test.firebaseapp.com',
+    databaseURL: 'https://coved-test.firebaseio.com',
+    projectId: 'coved-test',
+    storageBucket: 'coved-test.appspot.com',
+    messagingSenderId: '779651794617',
+    appId: '1:779651794617:web:8b9bf2f99882b58d337d68'
+};
```

Comment:
We should probably not commit our apiKey and other information.

Reply:
This stuff is fine to have here.